hungarian_algorithm_c.gemspec

```diff
@@ -0,0 +1,30 @@
+require_relative './lib/hungarian_algorithm_c/version'
+require 'date'
+
+Gem::Specification.new do |s|
+  s.name = 'hungarian_algorithm_c'
+  s.version = HungarianAlgorithmC::VERSION
+  s.date = Date.today.to_s
```

Comment:
Well... no, given the spec is "run" every time it's checked, the gem release date isn't "today" :)

Your gemspec looks really odd, too. Did you scaffold it with `bundle gem`? Or try to prep it manually?